x-pack/plugins/security_solution/public/management/components/console/service/parsed_command_input.ts

```diff
@@ -157,3 +157,15 @@ export const getArgumentsForCommand = (command: CommandDefinition): string => {
 
   return `${requiredArgs} ${optionalArgs.length > 0 ? `[${optionalArgs}]` : ''}`.trim();
 };
+
+export const parsedPidOrEntityIdParameter = (
```

Comment:
Should probably have some doc-comments on the business logic of what this represents. Why it's doing this (tho the fn name does that well in brief)